### ISSUE TITLE
Investigate saving logits in a second vector store

### ISSUE THREAD

**Issue by sdenton4** [google-research/perch-hoplite#119]
It's kind of annoying having a database of /either/ embeddings or logits for target species.

In theory we could make a second usearch vector store with the logits, and then have access to either/or. Ideally we could provide some interfaces for handling the logits without impacting existing databases.  This bug is to:
a) track trying out extending to a second usearch db for logits through vibe-coding,
b) document what kinds of changes to queries, etc would need to happen to support logits 'fluently.'
